### PR TITLE
Update content and styling

### PR DESCRIPTION
**Pull request by shrinathprabhu**
Resolves [AR-2816](https://team-1624093970686.atlassian.net/browse/AR-2816), [AR-2806](https://team-1624093970686.atlassian.net/browse/AR-2806) and [AR-2808](https://team-1624093970686.atlassian.net/browse/AR-2808)

## Changes

- Update hero section assets
- Make CTA buttons of equal size
- Add default styles for lists and links (in styles.css) and remove those styles from individual components
- Add subdescription to blurb component and update it in what's new section
- Update gradients and spacing overall
- Update overview description
- Update quicklinks description in get started section

## Screenshots
Screenshot from 2022-06-21 09-05-34
Screenshot from 2022-06-21 09-05-44
Screenshot from 2022-06-21 09-05-55
Screenshot from 2022-06-21 09-06-22

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -85,3 +85,29 @@ body {
...
+
+  @media (--viewport-small) {
+    font-size: 0.875rem !important;
+  }
+}
+
+a {
+  color: var(--color-blue);
+  text-decoration: none;
+}
```
Comment (shrinathprabhu) [inline assets/css/styles.css]: Add a new line

File: components/WhatsNewSection.vue [hunk 2]
```diff
@@ -57,7 +61,7 @@
         class="tablet-remove laptop-remove mobile-show"
       />
       <img
-        src="@/assets/images/features-gradient-bottom.png"
+        src="@/assets/images/features-gradient-bottom.svg"
```
Comment (shrinathprabhu) [inline components/WhatsNewSection.vue]: Try using css gradients

File: components/HeroSection.vue [hunk 3]
```diff
@@ -117,20 +130,20 @@ section {
 }
 
 .hero-section-container {
-  margin-top: 14vh;
+  margin-top: 6vw;
```
Comment (shrinathprabhu) [inline components/HeroSection.vue]: Use `vh`

File: components/HeroSection.vue [hunk 4]
```diff
@@ -117,20 +130,20 @@ section {
...
   float: right;
 
   @media (--viewport-small) {
-    margin-top: calc(((451 - 86) / 375) * 100vw + 10vh);
+    margin-top: calc(((451 - 86) / 375) * 100vw + 12vh);
```
Comment (shrinathprabhu) [inline components/HeroSection.vue:75]: @ajithranka What does this code snippet do?

File: components/QuickLinks.vue [hunk 5]
```diff
@@ -11,25 +11,16 @@
...
-            :label="quickLink.linkText"
-            label-transform="uppercase"
-            :href="quickLink.link"
-            target="_blank"
-          />
-        </v-stack>
+        <v-text
+          class="quicklinks-description"
+          line-height="1.5"
+          v-html="quickLink.description"
```
Comment (shrinathprabhu) [inline components/QuickLinks.vue:22]: Figure out if we can avoid using `v-html`

File: components/WhatsNewSection.vue [hunk 6]
```diff
@@ -39,9 +30,22 @@
           :alt="feature.title"
           :title="feature.title"
           :description="feature.description"
+          :sub-description="feature.subDescription"
```
Comment (shrinathprabhu) [inline components/WhatsNewSection.vue]: `description` becomes `caption`
and `sub-description` becomes `description`

File: components/WhatsNewSection.vue (hunk 2 above)
Comment (shrinathprabhu) [reply components/WhatsNewSection.vue]: Created ticket: https://team-1624093970686.atlassian.net/browse/AR-3147

File: components/QuickLinks.vue (hunk 5 above)
Comment (shrinathprabhu) [reply components/QuickLinks.vue:22]: Created a ticket: https://team-1624093970686.atlassian.net/browse/AR-3148

**Review by karthik-durai — APPROVED** [review]

File: assets/css/styles.css (hunk 1 above)
Comment (shrinathprabhu) [reply assets/css/styles.css]: Resolved at https://github.com/arcana-network/testnet-website/pull/22/commits/be8e05d426842b89fa579b7ea077e87dff3af49c

File: components/HeroSection.vue (hunk 3 above)
Comment (shrinathprabhu) [reply components/HeroSection.vue]: Resolved at https://github.com/arcana-network/testnet-website/pull/22/commits/a56fba5d13e62da3361088c9177f613ec5b430ee

File: components/WhatsNewSection.vue (hunk 6 above)
Comment (shrinathprabhu) [reply components/WhatsNewSection.vue]: Resolved at https://github.com/arcana-network/testnet-website/pull/22/commits/93581b64882442545b66492de037ab310dfd20ea

File: components/HeroSection.vue (hunk 4 above)
Comment (ajithranka) [reply components/HeroSection.vue:75]: It's trying to calculate the height of the background image to set the top margin on the hero section. 

The hero image is rendered at an aspect ration of 375 x 451 and the header at the top of the page takes 86px height. So:

```
margin-top = ((image height - header height) / (image width)) * 100vw + some fixed height
```

**Pull request merged**